hack/lib/etcd.sh

```diff
@@ -24,6 +24,8 @@ ETCD_PORT=${ETCD_PORT:-2379}
 # to the command line argument, even when both have the same value.
 ETCD_LOGLEVEL=${ETCD_LOGLEVEL:-warn}
 export KUBE_INTEGRATION_ETCD_URL="http://${ETCD_HOST}:${ETCD_PORT}"
+# ETCD_PROGRESS_NOTIFY_INTERVAL defines the interval for etcd watch progress notify events in seconds.
+ETCD_PROGRESS_NOTIFY_INTERVAL="${ETCD_PROGRESS_NOTIFY_INTERVAL:-5s}"
```

Comment:
I don't know what relies on this file.  Please find an owner to comment.  At the very least, test-infra and a googler from GKE.  These had weird dependencies in the past and I'd like them to know.

Reply:
@BenTheElder @cblecker perhaps one of you could help review this PR ?

Reply:
From GKE perspective we're good.

I don't know all the test-infra consequences.